system/jlib/jencrypt.cpp

```diff
@@ -1890,9 +1890,11 @@ MemoryBuffer &aesEncrypt(const void *key, size_t keylen, const void *plaintext,
     }
 }
 
-static void decryptError(const char *what)
+static void decryptError(const char *what, size32_t len=0)
```

Comment:
Its passed in always as a size_t, I suppose it doesn't really matter but why not use size_t len=0 here ?